R/gtrends.R

```diff
@@ -571,7 +586,7 @@ as.zoo.gtrends <- function(x, ...) {
     blocks <- lapply(start:length(vec), function(i)
       read.csv(
         textConnection(strsplit(vec[i], "\\\n")[[1]]),
-        skip = 1,
+        skip = 1, header = FALSE,
```

Comment:
You are right about Top.cities...But if we keep the header=TRUE (default), the first rows of the next 2 sections (top.searches and rising.searches) seem to take up the 1st query as the header as well. We will need to differentiate between top searches, rising searches and geographic information...probably using regex as proposed.

Considering the merge conflict as well, let me close this pull request and follow the methodical approach as proposed by @eddelbuettel in which we can propose/discuss one fix/enhancement per PR.